### PR TITLE
Add Unity supported operations for alert SNMP configurations:

### PR DESCRIPTION
**Pull request by DingDino**
-Collection query
-Instance query
-Create
-Delete
-Modify

### CODE REVIEW

**Comment by thedreadpirateeric** [comment]
The field `targetAddress` is showing up in the docs and the API as `address`. I have tested with the changes and it is workin as expected.

```
In [1]: from storops import UnitySystem

In [2]: uu = UnitySystem("IPADDRESS", "USERNAME", "PASSWORD")

In [3]: uu.get_alert_snmp_config()
Out[3]: /DIRLOCATION/storops/storops/lib/parser.py:360: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  ret = yaml.load(stream)
/DIRLOCATION/storops/storops/lib/metric.py:199: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  ret = yaml.load(stream)

{
    "UnityAlertConfigSNMPTargetList": [
        {
            "UnityAlertConfigSNMPTarget": {
                "address": "redactedhost.sb.redacted.com:162",
                "community": "UnityMessage",
                "existed": true,
                "hash": -9223372036567077930,
                "id": "snmp_target_11",
                "username": ""
            }
        }
    ]
}
```

File: storops/unity/parser_configs.yaml [hunk 1]
```diff
@@ -2227,3 +2227,20 @@ UnityImportSession:
     - label: cutoverThreshold
     - label: remainingDataPercent
     - label: estimatedCutoverWindow
+
+UnityAlertConfigSNMPTarget:
+  data_src: rest
+  name: alertConfigSNMPTarget
+  properties:
+    - label: id
+    - label: targetAddress
```
Comment (thedreadpirateeric) [inline storops/unity/parser_configs.yaml]: Should be `address`

File: storops/unity/resource/alert_config_snmp.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+LOG = logging.getLogger(__name__)
+
+
+class UnityAlertConfigSNMPTarget(UnityResource):
+
+    @classmethod
+    def create(cls, cli, target_address, auth_protocol=None, username=None,
+               auth_password=None, priv_protocol=None, priv_password=None,
+               snmp_version=None, community=None):
+        req_body = cli.make_body(targetAddress=target_address,
```
Comment (thedreadpirateeric) [inline storops/unity/resource/alert_config_snmp.py]: `targetAddress` should be `address`

File: storops/unity/resource/alert_config_snmp.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                                 community=community,
+                                 username=username)
+        resp = cli.post(cls().resource_class, **req_body)
+        resp.raise_if_err()
+        return cls.get(cli, resp.resource_id)
+
+    def modify(self, target_address=None, auth_protocol=None, username=None,
+               auth_password=None, priv_protocol=None, priv_password=None,
+               community=None):
+        req_body = self._cli.make_body(targetAddress=target_address,
```
Comment (thedreadpirateeric) [inline storops/unity/resource/alert_config_snmp.py]: `targetAddress` should be `address`

File: storops/unity/parser_configs.yaml (hunk 1 above)
Comment (DingDino) [reply storops/unity/parser_configs.yaml]: changed~
 
Thanks

File: storops/unity/resource/alert_config_snmp.py (hunk 2 above)
Comment (DingDino) [reply storops/unity/resource/alert_config_snmp.py]: I think this should be followed with create operation~ 
Use 'address' will gave 'wrong parameter ...' error

File: storops/unity/resource/alert_config_snmp.py (hunk 3 above)
Comment (DingDino) [reply storops/unity/resource/alert_config_snmp.py]: same with create()

File: storops/unity/resource/alert_config_snmp.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# Copyright (c) 2015 EMC Corporation.
```
Comment (Murray-LIANG) [inline storops/unity/resource/alert_config_snmp.py]: Change it to Copyright (c) 2019 Dell Inc. or its subsidiaries.

**Pull request merged**